ads/insticator.js

```diff
@@ -0,0 +1,115 @@
+// !!!!!!!!!! ---------- AMP EMBED - INSTICATOR ---------- !!!!!!!!!! //
+import { validateData, loadScript } from '../3p/3p';
+
+
+// ------- HELPER VARIABLES ------- //
+// reusable URL references to ads, embed and the library
+// Don't forget to preconnect and prefetch as it's described in the docs
+// https://github.com/ampproject/amphtml/blob/master/ads/README.md
+const url = {
+  content: '//d3lcz8vpax4lo2.cloudfront.net',
+  lib: 'https://testthisshit.online/amp-embed-lib/insticator.js'
+};
+
+
+// ------- HELPER FUNCTIONS ------- //
+const createInitialMarkup = (embedId) => {
+  return `
+    <div id="insticator-container">
+      <div id="div-insticator-ad-1"></div>
+      <div id="insticator-embed" embed-id="${embedId}"></div>
+      <div id="div-insticator-ad-2"></div>
+    </div>
+  `;
+};
+
+const createTemplate = (domString) => {
```

Comment:
can this be merged into `createInitialMarkup`

Reply:
Merged code